yarn.lock

```diff
@@ -4422,7 +4422,14 @@ axios-retry@^3.0.2:
   dependencies:
     is-retry-allowed "^1.1.0"
 
-axios@0.21.2, axios@^0.21.1, axios@^0.21.2:
+axios@0.21.2:
```

Comment:
Is this right? Looks like only axios changed in the lock file

Reply:
We recently added  a `resolve` to axios 0.21.2 because of a security vulnerability.
I imagine that we merged a PR without that resolve.